src/Exception/ConfigException.php

```diff
@@ -6,7 +6,7 @@
 
 use Exception;
 
-class ConfigException extends Exception
+final class ConfigException extends Exception
```

Comment:
Strictly speaking, this is a break in backward compatibility. What do you think about this @vjik?

Reply:
Yes, it breaks BC. In such cases I recommend:

- add `@final` annotation;
- add issue to mark class as final in next major version.